src/components/login-form/login-form.tsx

```diff
@@ -45,13 +45,10 @@ export default function LoginForm() {
                 type="email"
                 {...register("email", {
                   required: true,
-                  minLength: 6,
-                  maxLength: 30,
+                  pattern: /^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/,
```

Comment:
No need for patterns, Form component uses zod for validation and im also building validation schemas for backend using zod so you can leave out validation for me to fill